Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/interoperability/InteroperabilityRepository.kt

```diff
@@ -0,0 +1,29 @@
+package de.rki.coronawarnapp.storage.interoperability
+
+import de.rki.coronawarnapp.service.applicationconfiguration.ApplicationConfigurationService
+import de.rki.coronawarnapp.storage.LocalData
+import kotlinx.coroutines.runBlocking
+import java.util.Locale
+
+object InteroperabilityRepository {
+    private val TAG: String? = InteroperabilityRepository::class.simpleName
+
+    fun interoperabilityWasShown(): Boolean = LocalData.interoperabilityWasShown()
+
+    fun saveInteroperabilityUsed() {
+        LocalData.saveInteroperabilityUsed()
+    }
+
+    /**
+     * Gets all countries from @see ApplicationConfigurationService.asyncRetrieveApplicationConfiguration
+     * and filters out the CURRENT_COUNTRY from @see DiagnosisKeyConstants. Also changes every country code
+     * to lower case
+     */
+    fun getAllCountries(): List<String> {
```

Comment:
How about renaming the function to `getLConfiguredCountriesExceptCurrent()` and dropping the comment? Comments are rarely updated and this one describes the code line by line.

Reply:
The comment ist obsolete. Previously I filtered out the current country but since the UI changes it's kept in the list. Will change the comment 👍